index/server/main.go

```diff
@@ -512,3 +604,50 @@ func buildIndexAPIResponse(c *gin.Context, indexType string, iconType string) {
 		c.File(responseIndexPath)
 	}
 }
+
+// trackEvent tracks event for telemetry
+func trackEvent(event analytics.Message) error {
+	// Initialize client for telemetry
+	client := analytics.New(telemetryKey)
+	defer client.Close()
+
+	err := client.Enqueue(event)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+// getUser gets the user
+func getUser(c *gin.Context) string {
```

Comment:
Can we use a different request header than user? Maybe client or something like that?

Reply:
I also think we need to discuss how the registry clients will be (either via the go library or through rest) will be setting this header, and what the expected values will be.

Reply:
Sure, we can discuss that on part 2 which is client side change, this change is only for server side.

Reply:
The reason why we "user" here is to align with the terminology of Segment and Woopra, but discuss that later when I am working on the client side.